soda/scientific/soda/scientific/anomaly_detection/models/base.py

```diff
@@ -56,40 +56,30 @@ def _eliminate_measurements(
 
     def skip_measurements(self):
         """Eliminates measurements that are labelled as skipped by users."""
-        has_two_previouses = False
         is_inclusive = False
         max_index = []
-        if "skip_measurements" in self._data_df.columns:
+        if "skipMeasurements" in self._data_df.columns:
+            self._data_df = self._data_df.sort_values(by="ds", ascending=True)
+            self._data_df = self._data_df.reset_index(drop=True)
             skip_previous_inclusive_indexes = self._data_df.index[
-                self._data_df["skip_measurements"] == "previousAndThis"
+                self._data_df["skipMeasurements"] == "previousAndThis"
```

Comment:
instead of changing `skip_measurements` to `skipMeasurements` I would do the contrary. It's more pythonic.

Reply:
@baturayo yes totally agree! However, see this note in the PR description:

> during the sprint planning we discussed adding functionality to automatically change the camel case to snake case in the pydantic models, like we do in rca. However, since camel case is used in lots of other places during anomaly detection (not just in the skip_measurements method) I think it makes more sense to tackle them all in another ticket.

I think it makes sense to tackle all the camelcase stuff in another ticket

Reply:
indeed makes sense to tackle it in another ticket.